src/Parser/LASLoader.js

```diff
@@ -0,0 +1,144 @@
+import { LazPerf } from 'laz-perf';
+import { Las } from 'copc';
+
+/**
+ * @typedef {Object} Header - Partial LAS header.
+ * @property {number} header.pointDataRecordFormat - Type of point data
+ * records contained by the buffer.
+ * @property {number} header.pointDataRecordLength - Size (in bytes) of the
+ * point data records. If the specified size is larger than implied by the
+ * point data record format (see above) the remaining bytes are user-specfic
+ * "extra bytes". Those are described by an Extra Bytes VLR.
+ * @property {number[]} header.scale - Scale factors (an array `[xScale,
+ * yScale, zScale]`) multiplied to the X, Y, Z point record values.
+ * @property {number[]} header.offset - Offsets (an array `[xOffset,
+ * xOffset, zOffset]`) added to the scaled X, Y, Z point record values.
+ */
+
+class LASLoader {
```

Comment:
Does it works for las and laz seamlessly ? If yes, indicate it in the class doc and modify comment of `parse` method

Reply:
Done.